webview/src/plots/components/PlotsContainer.tsx

```diff
@@ -32,7 +34,9 @@ export interface PlotsContainerProps extends CommonPlotsContainerProps {
   sectionKey: Section
   title: string
   currentSize: number
-  menu?: PlotsPickerProps
+  selectPlotsMenu?: PlotsPickerProps
```

Comment:
With me adding multiple items, I wanted to make it more clear on the difference between the items `menu` has a tooltip name of `Select Plots` so it can't currently be used for anything else.  I can change it bad to `menu` if preferred though!

Reply:
We removed the sizing from the menu, but the reset size should go back inside that menu. Same if we get other actions.

Reply:
Apologies, I'm a little confused about your comment. What do you mean by "sizing"? And how is it related to the name change?

Reply:
Before there was a button insde the menu for the size (instead of resizing the plot directly) it was removed, but it is planned to re-add a 'Reset plots size' inside that menu. Changing `menu` to `selectPlotsMenu` makes it less extensible to other actions unrelated to plot selection.

Reply:
Oh! Thanks for the clarification, I'll change it back to `menu` :)